src/jwt/mod.rs

```diff
@@ -20,11 +20,98 @@ new_type![
     JsonWebTokenContentType(String)
 ];
 
+#[derive(Error, Debug)]
+pub enum JsonWebTokenTypeNormalizeError {
+    #[error("Has parameters set without application specifies like JWT;param=true (should be application/JWT;param=true)")]
+    ParameterWithoutApplication(),
+
+    #[error(
+        "The web token type is complex (contains /) but does not start with an application type."
+    )]
+    ComplexTypeWithoutApplication(),
+}
+
 new_type![
+    /// JSON web token type field (typ)
```

Comment:
`web token` -> `Web Token`

Reply:
There is one more "web token" in this file. Should I fix this too (maybe in an extra commit)?